src/main/java/it/pagopa/transactions/commands/handlers/TransactionSendClosureHandler.java

```diff
@@ -88,8 +89,15 @@ public Mono<TransactionClosureSentEvent> handle(TransactionClosureSendCommand co
 
                                 TransactionClosureSentEvent event = new TransactionClosureSentEvent(
                                         command.getData().transaction().getTransactionId().value().toString(),
-                                        command.getData().transaction().getRptId().value(),
-                                        command.getData().transaction().getTransactionActivatedData().getPaymentToken(),
+                                        command.getData().transaction().getNoticeCodes().stream().map(
+                                                noticeCode ->  new NoticeCode(
+                                                        command.getData().transaction().getTransactionActivatedData().getNoticeCodes().
+                                                                stream().filter(noticeCode1 -> noticeCode1.getRptId().equals(noticeCode.rptId().value())).findFirst().get()
+                                                                .getPaymentToken(),
```

Comment:
same as src/main/java/it/pagopa/transactions/commands/handlers/TransactionAddUserReceiptHandler.java